### PR TITLE
ROR-13 Deploy Scheduling API

### PR DESCRIPTION
**Pull request by jamiejordanmt**
[ROR-13 Deploy Scheduling API](https://hackney.atlassian.net/browse/ROR-13)

Converted from GitHub actions to CircleCi
Used Serverless framework to deploy to AWS Lambda
Used AWS Parameter Store for configuration (environment variables)
Upgraded from .Net 5.0 to 6.0
Disabled external integrations and temporarily used dummy replacements where appropriate.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    parameters:
+      lincoln-nuget-username:
+        type: string
+        default: $LINCOLN_NUGET_USERNAME
+      lincoln-nuget-password:
+        type: string
+        default: $LINCOLN_PACKAGES_TOKEN
+    steps:
+      - run:
+          name: Add HACT package
```
Comment (Bhavesh-Patel) [inline .circleci/config.yml]: This is for more than just HACT package, it's authentication too.

Please replace the name with something generic.

File: HousingRepairsSchedulingApi/Properties/launchSettings.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
         "ASPNETCORE_ENVIRONMENT": "Development",
         "AUTHENTICATION_IDENTIFIER": "<REPLACE WITH A UNIQUE AUTHENTICATION IDENTIFIER>",
         "JWT_SECRET" : "<REPLACE WITH A SECRET>",
-        "DrsOptions__ApiAddress" : "https://<REPLACE WITH DRS API ADDRESS>",
+        "DrsOptions__ApiAddress" : "https://www.bbc.co.uk",
```
Comment (Bhavesh-Patel) [inline HousingRepairsSchedulingApi/Properties/launchSettings.json]: Please restore to generic "replace with..." value.

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]
Couple of minor changes.

File: .circleci/config.yml (hunk 1 above)
Comment (jamiejordanmt) [reply .circleci/config.yml]: @Bhavesh-Patel I've updated the name - is this ok, or do you have a better suggestion?

Comment (Bhavesh-Patel) [reply .circleci/config.yml]: Yes, that's fine.

**Review by Bhavesh-Patel — APPROVED** [review]

**Pull request merged**